.github/CONTRIBUTING.md

```diff
@@ -67,6 +68,21 @@ Enter password for new role: peertube
 # sudo -u postgres createdb -O peertube peertube_dev
 ```
 
+You also need to create a system user for Peertube:
```

Comment:
Why did you need this peertube user? You should be able to dev on peertube with your regular user

Reply:
Ah I misunderstood something, will remove this.